app/move/controllers/create/base.js

```diff
@@ -20,7 +20,8 @@ class CreateBaseController extends FormWizardController {
 
   checkCurrentLocation(req, res, next) {
     if (!req.session.currentLocation) {
-      const error = new Error('Current location is not set in session.')
+      const error = new Error('Current location is not set in session')
+      error.code = 'MISSING_LOCATION'
```

Comment:
Maybe for future but these could sit in one place?

Reply:
The errors? Possibly, although this one is specific to this part of the app. The others are set within the actual form wizard library too so not sure how easy it would be to extract them all and merge them.